src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -328,15 +336,15 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
 
               <TokenExtras>
                 {amountValue && <StyledText>{formatUsd(!loading && !inputError ? amountValue : '0')}</StyledText>}
-                {yieldPercent && (
-                  <StyledText>
-                    {t('components.transaction.token-input.yield')} <ContrastText>{yieldPercent}</ContrastText>
-                  </StyledText>
-                )}
               </TokenExtras>
             </TokenData>
           )}
         </TokenInfo>
+        {listItems?.length > 1 && (
+          <ZapMessageContainer onClick={openSearchList}>
+            ⚡ Or <HighlightText> select a token </HighlightText> in your wallet to ZAP
```

Comment:
Text should be moved to locales so it can also be translated

Reply:
i don't know how to do this - can i ask for help again please - sorry

Reply:
sure, added here https://github.com/yearn/yearn-finance-v3/pull/690/commits/09db4c7965666ca6b14b7df93ab5db6199adee8c :)